internal/cassandra.go

```diff
@@ -41,6 +41,16 @@ type Database interface {
 	RemoveFromTagIndex(tagKey, tagValue string, metricKey api.MetricKey) error
 }
 
+type DatabaseGraphiteStore interface {
```

Comment:
It's not really necessary (we can get rid of the hierarchy by dropping the embed and renaming it to `GraphiteStore`).